listenbrainz_spark/ftp/__init__.py

```diff
@@ -20,7 +19,7 @@ def connect(self):
             self.connection = ftplib.FTP(config.FTP_SERVER_URI)
             self.connection.login()
         except ftplib.error_perm:
-            current_app.logger.critical("Couldn't connect to FTP Server, try again...")
+            logging.critical("Couldn't connect to FTP Server, try again...")
```

Comment:
if we are going to use python logging outside of flask then we should make some effort to configure it.
It'd be good to be able to set a level in config.py. Additionally, we should have a logging setup method that allows us to configure the handlers that we want (and the ability to add a file handler if needed). We could put this in a separate py file (or listenbrainz_spark/init) and use it in all places that we do logging

Reply:
That's a great idea! I'll get to it.